modules/openapi-generator/src/main/resources/r/model.mustache

```diff
@@ -181,24 +181,32 @@
            {{/vars}}
         }',
         {{#vars}}
+        {{! Added the if else condition to handle null values}}
+        if (!is.null(self$`{{baseName}}`)) {
         {{#isListContainer}}
         {{#isPrimitiveType}}
-        paste(unlist(lapply(self$`{{{baseName}}}`, function(x) paste0('"', x, '"'))), collapse=","){{#hasMore}},{{/hasMore}}
+        paste(unlist(lapply(self$`{{{baseName}}}`, function(x) paste0('"', x, '"'))), collapse=",")
         {{/isPrimitiveType}}
         {{^isPrimitiveType}}
-        paste(unlist(lapply(self$`{{{baseName}}}`, function(x) jsonlite::toJSON(x$toJSON(), auto_unbox=TRUE))), collapse=","){{#hasMore}},{{/hasMore}}
+        paste(unlist(lapply(self$`{{{baseName}}}`, function(x) jsonlite::toJSON(x$toJSON(), auto_unbox=TRUE))), collapse=",")
         {{/isPrimitiveType}}
         {{/isListContainer}}
         {{^isListContainer}}
         {{#isPrimitiveType}}
-        self$`{{baseName}}`{{#hasMore}},{{/hasMore}}
+        self$`{{baseName}}`
         {{/isPrimitiveType}}
         {{^isPrimitiveType}}
-        jsonlite::toJSON(self$`{{baseName}}`$toJSON(), auto_unbox=TRUE){{#hasMore}},{{/hasMore}}
+        jsonlite::toJSON(self$`{{baseName}}`$toJSON(), auto_unbox=TRUE)
         {{/isPrimitiveType}}
         {{/isListContainer}}
+	}
+	else{
+	  "null"
+	}{{#hasMore}},{{/hasMore}}
         {{/vars}}
       )
+	  outstring = gsub("[\r\n]| ", "", outstring)
```

Comment:
I'm afraid this won't work if the data/value contains line breaks

Reply:
@wing328 you are right, i can remove this line considering what you mentioned. Thanks